src/attribute.rs

```diff
@@ -815,6 +860,12 @@ impl From<String> for AttrValue {
     }
 }
 
+impl From<Vec<String>> for AttrValue {
```

Comment:
Note for future: This can be made more general using a different type, e.g. `AsRef<[&str]>`

Reply:
Running into problems with `AsRef` (upstream may add a new impl of trait), the following two impls should along with the two you've written cover most of the usecases:
```rust
impl From<&[&str]> for AttrValue {
    fn from(x: &[&str]) -> Self {
        Self::Strs(x.iter().map(|&s| String::from(s)).collect())
    }
}

impl From<Vec<&str>> for AttrValue {
    fn from(x: Vec<&str>) -> Self {
        Self::from(x.as_slice())
    }
}
```

Reply:
Added